runtime/doc/api.txt

```diff
@@ -866,32 +866,33 @@ nvim_get_api_info()                                      *nvim_get_api_info()*
                     {fast}
 
 nvim_get_chan_info({chan})                              *nvim_get_chan_info()*
-                Get information about a channel.
+                Gets information about a channel.
```

Comment:
Just as a general question: should docs be in the imperative or descriptive voice? I've seen both, but it would probably be good to be consistent.

Reply:
descriptive for docstrings . imperative  usually makes sense for comments.

I once read a good argument for this distinction, but didn't know the term "descriptive", thanks. Essentially, code comments, if they are necessary, describe a procedure or implementation. Whereas docstrings describe the interface.

By using the descriptive form, it avoids the tendency to put implementation details in docstrings. I think?